Chargebee/Classes/ReceiptValidation/CBValidateReceiptResource.swift

```diff
@@ -83,7 +86,10 @@ struct PayloadBody: URLEncodedRequestBody {
             "product[currency_code]": currencyCode,
             "product[period]": period,
             "product[period_unit]": periodUnit,
-            "customer[id]": customerId
+            "customer[id]": customerId,
+            "first_name": first_name,
+            "last_name": last_name,
+            "email": email
```

Comment:
This should be `customer[{}]`

Reply:
yes updated once testing is done will push  addressed changes to PR.